psi4/src/psi4/occ/arrays.h

```diff
@@ -141,41 +135,6 @@ class Array2d {
     void lineq_pople(Array1d* Xvec, int num_vecs, double cutoff);
     // gemm: matrix multiplication
     void gemm(bool transa, bool transb, double alpha, const Array2d* a, const Array2d* b, double beta);
-    // level_shift: A[i][i] = A[i][i] - value
-    void level_shift(double value);
-    // outer_product: A = x * y'
-    void outer_product(const Array1d* x, const Array1d* y);
-    void scale(double a);
-    // scale_row: scales mth row with a
-    void scale_row(int m, double a);
-    // scale_column: scales nth column with a
-    void scale_column(int n, double a);
-    // identity: A = I
-    void identity();
-    double trace();
-    // transform: A = L' * B * L
-    void transform(const Array2d* a, const Array2d* transformer);
-    // back_transform: A = L * B * L'
-    void back_transform(const Array2d* a, const Array2d* transformer);
-    // pseudo_transform: A = L * B * L
-    void pseudo_transform(const Array2d* a, const Array2d* transformer);
-    // triple_gemm: A2d_ = a * b * c
-    void triple_gemm(const Array2d* a, const Array2d* b, const Array2d* c);
-    // vector_dot: value = Tr(A' * B)
-    double vector_dot(Array2d* rhs);
-    double vector_dot(double** rhs);
-    double** to_block_matrix();
-    double* to_lower_triangle();
-    // mgs: orthogonalize with a Modified Gram-Schmid algorithm
-    void mgs();
-    // gs: orthogonalize with a Classical Gram-Schmid algorithm
-    void gs();
-    // row_vector: return nth row as a vector
```

Comment:
I see those in use: https://github.com/psi4/psi4/blob/master/psi4/src/psi4/dfocc/update_hfmo.cc#L85-L88

Reply:
oh, this is `dfocc` not `occ`.